### ISSUE TITLE
[V3] Help crashes when giving a command which doesn't exist

### ISSUE THREAD

**Issue by ZeLarpMaster** [Cog-Creators/Red-DiscordBot#1766]
# Command bugs

#### Command name

[p]help

#### What cog is this command from?

No cog (`help_formatter.py` I guess)

#### What were you expecting to happen?

"No command called "{command}" found." as in V2 or something similar

#### What actually happened?

The bot replied with:  
CommandInvokeError: Command raised an exception: AttributeError: 'coroutine' object has no attribute 'to_dict'  
`Error in command 'help'. Check your console or logs for details.`

There was the following traceback in the console:
```
  File "<python path>\lib\site-packages\discord\ext\commands\core.py", line 62, in wrapped
    ret = yield from coro(*args, **kwargs)
  File "<python path>\lib\site-packages\redbot\core\help_formatter.py", line 330, in help
    await destination.send(embed=await ctx.bot.formatter.cmd_not_found(ctx, name))
  File "<python path>\lib\site-packages\discord\abc.py", line 737, in send
    embed = embed.to_dict()
[<datetime>] ERROR events on_command_error 206: Exception in command 'help'
Traceback (most recent call last):
  File "<python path>\lib\site-packages\discord\ext\commands\core.py", line 62, in wrapped
    ret = yield from coro(*args, **kwargs)
  File "<python path>\lib\site-packages\redbot\core\help_formatter.py", line 330, in help
    await destination.send(embed=await ctx.bot.formatter.cmd_not_found(ctx, name))
  File "<python path>\lib\site-packages\discord\abc.py", line 737, in send
    embed = embed.to_dict()
AttributeError: 'coroutine' object has no attribute 'to_dict'
<python path>\lib\asyncio\events.py:127: RuntimeWarning: coroutine 'Help.simple_embed' was never awaited
  self._callback(*self._args)
```

#### How can we reproduce this issue?

1. Install `Red - Version 3.0.0b15-2`
2. Start the bot with a valid token and join a server with the bot
3. Do `[p]help an invalid command`

**Comment by ZeLarpMaster** [comment]
Note: I talked about this issue previously, but I don't think anyone tried to fix it. I'm mostly opening an issue to make sure it gets fixed.

**Comment by Tobotimus** [comment]
Thanks for raising this issue, but it has actually been fixed, and it's a duplicate of #1746 - unfortunately the fix didn't make it into `3.0.0b15.post2`.
